src/jit/Compiler.h

```diff
@@ -176,6 +178,11 @@ union InstructionValue {
     size_t targetLabelCount;
     // For immediate and local groups.
     Instruction* parent;
+    // For memory instructions.
+    struct {
+        uint32_t offset;
+        uint32_t alignment;
```

Comment:
Don't add two members for this. Alignment should be stored in `info()` as bits, since its value is quite limited.